### ISSUE TITLE
Consider re-enabling cargo-readme

### ISSUE THREAD

**Issue by heaths** [Azure/azure-sdk-for-rust#1643]
Running cargo-readme on #1642 breaks due to webern/cargo-readme#81. Consider re-enabling it in the future, though I don't think long-term we'll want to generate any part of our READMEs. We don't for other languages, and READMEs should follow the same format.

**Comment by RickWinter** [comment]
We should use the usual pattern for creating readmes. @heaths is there any value in keeping this issue open?

**Comment by heaths** [comment]
No, not any more. Now that I have guidelines on samples in READMEs and with following the similar structure as other repos, we don't need this anymore.
